.pre-commit-config.yaml.jinja

```diff
@@ -104,40 +103,45 @@ repos:
       - id: mixed-line-ending
         args: ["--fix=lf"]
   - repo: https://github.com/pycqa/flake8
-    rev: 3.8.3
+    rev: 5.0.4
     hooks:
       - id: flake8
-        name: flake8 except __init__.py
-        exclude: /__init__\.py$
-        additional_dependencies: ["flake8-bugbear==20.1.4", "importlib-metadata<5.0.0"]
+        name: flake8 except __init__.py and __manifest__.py
+        exclude: /__(?:init|manifest)__\.py$
+        additional_dependencies: ["flake8-bugbear==23.3.12", "importlib-metadata==4.2.0"]
       - id: flake8
         name: flake8 only __init__.py
         args: ["--extend-ignore=F401"] # ignore unused imports in __init__.py
         files: /__init__\.py$
-        additional_dependencies: ["flake8-bugbear==20.1.4", "importlib-metadata<5.0.0"]
+        additional_dependencies: ["flake8-bugbear==23.3.12", "importlib-metadata==4.2.0"]
+      - id: flake8
+        name: flake8 only __manifest__.py
+        args: ["--extend-ignore=B018"] # ignore useless expressions in __manifest__.py
+        files: /__manifest__\.py$
+        additional_dependencies: ["flake8-bugbear==23.3.12", "importlib-metadata==4.2.0"]
```

Comment:
Instead of having the double hook, what do you think about having a single hook and adding at the end of `.flake8` 

```
per-file-ignores=
    __init__.py:F401
    __manifest__.py:B018
```
?